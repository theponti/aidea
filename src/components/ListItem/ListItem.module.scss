@import "../../styles/colors.module.scss";

.container {
  padding: 1rem;
  border-radius: 3px;
  // background-color: $card-background;
  margin-bottom: 1rem;
  box-shadow: 0px 1px 12px 2px #e8e8e8;
}
