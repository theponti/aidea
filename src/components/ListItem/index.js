import t from "prop-types";
import React from "react";
import styles from "./ListItem.module.scss";

export function ListItem({ children }) {
  return <div className={styles.container}>{children}</div>;
}

ListItem.propTypes = {
  children: t.node,
};

export default ListItem;
