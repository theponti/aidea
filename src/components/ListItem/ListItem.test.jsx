import { render } from "@testing-library/react";
import { describe, expect, it } from "vitest";
import ListItem from ".";

describe("<ListItem/>", () => {
  it("should render", () => {
    const { container } = render(<ListItem />);
    expect(container).toMatchSnapshot();
  });
});
