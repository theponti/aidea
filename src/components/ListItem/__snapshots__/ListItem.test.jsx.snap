// Vitest Snapshot v1

exports[`<ListItem/> > should render 1`] = `
<div>
  <div
    class="_container_1fxe8_1"
  />
</div>
`;
